syntax = "proto3";

service UserService {
    rpc RemoveAndBlockUser (UserRequest) returns (UserResponse);
}

message UserRequest {
    string user_id = 1;
}

message UserResponse {
    string message = 1;
}